opa/stackable/bin/process-logs

```diff
@@ -0,0 +1,139 @@
+# and forwarded to the output destinations console and/or file.
+#
+
+parse_args() {
+  while true; do
+    case $1 in
+        --console-log-level)
+            shift
+            CONSOLE_LEVEL=$1
+            ;;
+        --file-log-level)
+            shift
+            FILE_LEVEL=$1
+            ;;
+        --decision-log-level)
+            shift
+            DECISION_LEVEL=$1
+            ;;
+        --server-log-level)
+            shift
+            SERVER_LEVEL=$1
+            ;;
+        --opa-rolling-log-file-size-bytes)
+            shift
+            OPA_ROLLING_LOG_FILE_SIZE_BYTES=$1
+            ;;
+        --opa-rolling-log-files)
+            shift
+            OPA_ROLLING_LOG_FILES=$1
+            ;;
+        --stackable-log-dir)
+            shift
+            STACKABLE_LOG_DIR=$1
+            ;;
+        --container-name)
+            shift
+            CONTAINER_NAME=$1
+            ;;
+        *)
+            break
+            ;;
+    esac
+    shift
+  done
+
+  if [ -z "${CONSOLE_LEVEL}" ]; then
+    echo "ERROR: parameter --console-log-level cannot be empty"
+    exit 1
+  fi
+
+  if [ -z "${FILE_LEVEL}" ]; then
+    echo "ERROR: parameter --file-log-level cannot be empty"
+    exit 1
+  fi
+
+  if [ -z "${DECISION_LEVEL}" ]; then
+    echo "ERROR: parameter --decision-log-level cannot be empty"
+    exit 1
+  fi
+
+  if [ -z "${SERVER_LEVEL}" ]; then
+    echo "ERROR: parameter --server-log-level cannot be empty"
+    exit 1
+  fi
+
+  if [ -z "${OPA_ROLLING_LOG_FILE_SIZE_BYTES}" ]; then
+    echo "ERROR: parameter --opa-rolling-log-file-size-bytes cannot be empty"
+    exit 1
+  fi
+
+  if [ -z "${OPA_ROLLING_LOG_FILES}" ]; then
+    echo "ERROR: parameter --opa-rolling-log-files cannot be empty"
+    exit 1
+  fi
+
+  if [ -z "${STACKABLE_LOG_DIR}" ]; then
+    echo "ERROR: parameter --stackable-log-dir cannot be empty"
+    exit 1
+  fi
+
+  if [ -z "${CONTAINER_NAME}" ]; then
+    echo "ERROR: parameter --container-name cannot be empty"
+    exit 1
+  fi
+}
+
+get_levels() {
+    case $1 in
+        fatal)
+            echo '["fatal"]' ;;
+        error)
+            echo '["error","fatal"]' ;;
+        warn)
+            echo '["warn","error","fatal"]' ;;
+        info)
+            echo '["info","warn","error","fatal"]' ;;
+        debug)
+            echo '["debug","info","warn","error","fatal"]' ;;
+        trace)
+            echo '["trace","debug","info","warn","error","fatal"]' ;;
+        *)
+            echo '[""]' ;;
+    esac
+}
+
+main() {
+    parse_args $@
+
+    local DECISION_LEVELS=$(get_levels $DECISION_LEVEL)
+    local SERVER_LEVELS=$(get_levels $SERVER_LEVEL)
+    local CONSOLE_LEVELS=$(get_levels $CONSOLE_LEVEL)
+    local FILE_LEVELS=$(get_levels $FILE_LEVEL)
+
+    jq -c --unbuffered 'if .decision_id then .logger = "decision" else .logger = "server" end' |
+    jq -c --unbuffered --arg decision_levels $DECISION_LEVELS --arg server_levels $SERVER_LEVELS \
```

Comment:
Any reason to use separate jqs for these, rather than just piping inside of jq?

Reply:
Actually, thinking about it a bit more, this feels like exactly the sort of remapping we'd use Vector VRL for.

Reply:
Given that it's all going into the file _after_ processing and that it's going into Vector afterwards anyway, yes.

On the other hand, we'd probably want to address https://github.com/stackabletech/operator-rs/issues/794 first, so fine to punt on it for now.